README.md

```diff
@@ -1,26 +1,33 @@
-[![](https://jitpack.io/v/spotify/openfeature-kotlin-sdk.svg)](https://jitpack.io/#spotify/openfeature-kotlin-sdk)
+<!-- markdownlint-disable MD033 -->
+<p align="center">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/open-feature/community/0e23508c163a6a1ac8c0ced3e4bd78faafe627c7/assets/logo/horizontal/white/openfeature-horizontal-white.svg">
+    <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/open-feature/community/0e23508c163a6a1ac8c0ced3e4bd78faafe627c7/assets/logo/horizontal/black/openfeature-horizontal-black.svg">
+    <img align="center" alt="OpenFeature Logo">
+  </picture>
+</p>
 
-# OpenFeature Kotlin SDK
+<h2 align="center">OpenFeature Kotlin SDKs</h2>
 
 ![Status](https://img.shields.io/badge/lifecycle-alpha-a0c3d2.svg)
 
-What is OpenFeature?
+## 👋 Hey there! Thanks for checking out the OpenFeature Kotlin SDK
+
+### What is OpenFeature?
+
 [OpenFeature][openfeature-website] is an open standard that provides a vendor-agnostic, community-driven API for feature flagging that works with your favorite feature flag management tool.
 
-Why standardize feature flags?
-Standardizing feature flags unifies tools and vendors behind a common interface which avoids vendor lock-in at the code level. Additionally, it offers a framework for building extensions and integrations and allows providers to focus on their unique value proposition.
+### Why standardize feature flags?
 
-This Kotlin implementation of an OpenFeature SDK has been developed at Spotify, and currently made available and maintained within the Spotify Open Source Software organization. Part of our roadmap is for the OpenFeature community to evaluate this implementation and potentially include it in the existing ecosystem of [OpenFeature SDKs][openfeature-sdks].
+Standardizing feature flags unifies tools and vendors behind a common interface which avoids vendor lock-in at the code level. Additionally, it offers a framework for building extensions and integrations and allows providers to focus on their unique value proposition.
 
-## Requirements
+## 🔍 Requirements
 
 - The Android minSdk version supported is: `21`.
 
 Note that this library is intended to be used in a mobile context, and has not been evaluated for use in other type of applications (e.g. server applications).
 
-## Usage
-
-### Adding the library dependency
+## 📦 Installation
```

Comment:
Just a suggestion: we can have a sub-section titles `Jitpack` (with the current instructions) and then add a new `Maven` section that we mark as "Work In Progress" or something like this

Reply:
Sounds good!